src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -240,8 +242,8 @@
             </Style>
         </ResourceDictionary>
     </UserControl.Resources>
-    <Grid>
 
+    <Grid Name ="inputGrid" MinHeight="100" MinWidth="100">
```

Comment:
Would this change the current behavior?

Reply:
No, this will only set the min height and width so that when resizing the node, the thumb icon doesn't get collapsed.